torch/nn/modules/distance.py

```diff
@@ -7,13 +7,21 @@
 
 class PairwiseDistance(Module):
     r"""
-    Computes the pairwise distance between vectors :math:`v_1`, :math:`v_2` using the p-norm:
+    Computes the pairwise distance between input vectors, or between columns of input matrices.
+
+    Distances are computed using ``p``-norm, with constant ``eps`` added to avoid division by zero
+    if ``p`` is negative, i.e.:
 
     .. math ::
-        \Vert x \Vert _p = \left( \sum_{i=1}^n  \vert x_i \vert ^ p \right) ^ {1/p}.
+        \mathrm{dist}\left(\mathbf{v}, \mathbf{u}\right) = \left\Vert \mathbf{v}-\mathbf{u} + \epsilon \mathbf{e} \right\Vert_p,
+
+    where :math:`\mathbf{e}` is the vector of ones and the ``p``-norm is given by.
+
+    .. math ::
+        \Vert \mathbf{x} \Vert _p = \left( \sum_{i=1}^n  \vert x_i \vert ^ p \right) ^ {1/p}.
```

Comment:
Why `mathbf`? Is this physics notation for vectors? We don't use that notation in PyTorch really.

Reply:
I guess it is a physics thing, i.e., bold symbols are vectors.
What do is used for vectors then? `\vec{x}`, or just `x`? I tried finding an example in the docs, but it is hard to search for and I couldn't find anything.

Reply:
We just use `x`. For example, see https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html

In general, in the linalg docs, I used capital letters for matrices (usually `A, B`) and small letters for vectors (usually `x`, `y`).